src/App.vue

```diff
@@ -22,8 +22,15 @@ export default {
   },
   data() {
     return{
-      loggedIn: false,
-      currentUser: {}
+      loggedIn: true,
```

Comment:
Good idea making the login default true so that we dont have to log in every time we refresh.